lib/clouds/gce_cloud_ops.py

```diff
@@ -718,6 +768,29 @@ def is_vm_ready(self, obj_attr_list) :
             obj_attr_list["last_known_state"] = "not running"
             return False    
 
+    @trace
+    def vm_placement(self, obj_attr_list) :
+        '''
```

Comment:
Please remove functions like this and define them in the parent class if they don't have an alternate default behavior.

Reply:
I wholeheartedly agree with you aim, but I would like to propose that we do that - for all adapters - in a separate PR. Fair? If you agree, I will take care of it (thankfully, we have a robust way to test functionality on ALL clouds now).

Reply:
sure thing.